reproduction-dependencies/AppDomain.Instance/AppDomainInstanceProgram.cs

```diff
@@ -59,6 +59,7 @@ public int Run(string[] args)
                 return -10;
             }
 
+            Console.ReadKey();
```

Comment:
Not a huge deal but this will block tests, also, don't they stay open by default now?

Reply:
I was debugging. That should have been removed. Taking it out.